### PR TITLE
Document contexts feature

### PR DESCRIPTION
**Pull request by tashian**

### CODE REVIEW

File: src/pages/docs/step-cli/the-step-command.mdx [hunk 1]
```diff
@@ -41,6 +41,38 @@ The file `$STEPPATH/config/defaults.json` can contain a JSON object listing defa
     It is not currently possible to segment default keys in <Code>defaults.json</Code> by subcommand. All keys are applied to all <Code>step</Code> commands.
 </Alert>
 
+## Working with multiple CAs
+
+By default, the `step` toolchain assumes a single CA is in use, and it will store that CA's configuration and state directories (`config`, `certs`, `secrets`, `db`, etc.) inside a single folder, `$HOME/.step`.
+
+If you regularly work with multiple CAs, we recommend enabling _contexts_.
+
+Contexts let you configure and select multiple CAs and configuration profiles. If this optional feature is enabled, the `$HOME/.step` folder will have a different layout.
```
Comment (dopey) [inline src/pages/docs/step-cli/the-step-command.mdx]: Let's use the separate sentence or thought structure you had mentioned before.

File: src/pages/docs/step-cli/the-step-command.mdx [hunk 2]
```diff
@@ -41,6 +41,38 @@ The file `$STEPPATH/config/defaults.json` can contain a JSON object listing defa
...
+
+Every context consists of a configuration profile directory (which includes user-managed [`defaults.json`](#configuration-file) and `ca.json`), and a directory for CA-managed data (CA roots, certificates, secrets, templates).
```
Comment (dopey) [inline src/pages/docs/step-cli/the-step-command.mdx]: There is no user managed ca.json. Only the defaults.json. There's only one `ca.json` and it's under the authorities. I could see why someone might think that's "user-managed" though.

File: src/pages/docs/step-cli/the-step-command.mdx [hunk 3]
```diff
@@ -41,6 +41,38 @@ The file `$STEPPATH/config/defaults.json` can contain a JSON object listing defa
...
+The root certificate has been saved in /home/carl/.step/authorities/ca/certs/root_ca.crt.
+The authority configuration has been saved in /home/carl/.step/authorities/ca/config/defaults.json.
+The profile configuration has been saved in /home/carl/.step/profiles/ca/config/defaults.json.
+$ step context current
+vpn
+```
+
+When contexts are enabled:
+- User configurations profiles are now stored in the `profiles` directory, and CA-managed data is stored in `authorities` directory.
+- Context configuration files `contexts.json` and `current-context.json` are created.
```
Comment (dopey) [inline src/pages/docs/step-cli/the-step-command.mdx]: and stored at the top level of the $STEPPATH (~/.step).

**Review by dopey — APPROVED** [review]
Overall, this is great. Made a couple of small recommendations. And pointed out one bit that was incorrect (`ca.json` is not in the profiles directory structure, but rather the authorities).

File: src/pages/docs/step-cli/the-step-command.mdx (hunk 1 above)
Comment (tashian) [reply src/pages/docs/step-cli/the-step-command.mdx:50]: sorry, can you elaborate?

**Pull request merged**